### ISSUE TITLE
[BUG] python-fastapi: pytest fails

### ISSUE THREAD

**Issue by Minyus** [OpenAPITools/openapi-generator#10280]
##### Description

pytest fails as follows:

```
#16 0.795 =================================== FAILURES ===================================
#16 0.795 _______________________________ test_place_order _______________________________
#16 0.795
#16 0.795 client = <starlette.testclient.TestClient object at 0x7f085de03210>
#16 0.795
#16 0.795 def test_place_order(client: TestClient):
#16 0.795 """Test case for place_order
#16 0.795
#16 0.795 Place an order for a pet
#16 0.795 """
#16 0.795 > body = openapi_server.Order()
#16 0.795 E NameError: name 'openapi_server' is not defined
#16 0.795
#16 0.795 tests/test_store_api.py:69: NameError

```

##### openapi-generator version

5.2.1

##### OpenAPI declaration file content or url

https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/3_0/petstore.yaml

##### Generation Details

Run: `docker run --rm -v ${PWD}:/local openapitools/openapi-generator-cli:v5.2.1 generate -i /local/openapi.yaml -g python-fastapi -o /local`

##### Steps to reproduce

Run: `docker-compose up` (or `docker-compose build --no-cache`)

##### Related issues/PRs

None

##### Suggest a fix

Remove `openapi_server.` and add `.dict()` in the generated test code, for example:

Original:
```
    body = openapi_server.Order()
```
Modification:
```
    body = Order().dict()
```

**Comment by wing328** [comment]
@Minyus thanks for reporting the issue. Can you please give it another try with the latest stable version v5.3.0 to see if the issue still occurs?

**Comment by Minyus** [comment]
@wing328 
Yes, the latest version released yesterday (https://hub.docker.com/layers/openapitools/openapi-generator-cli/latest-release/images/sha256-ab4fca118fb195b261956df22a04854a593c13f9b35f2e24572cc083f827ef35?context=explore) still reproduced the same symptom.

The generation code I tried was: 
`docker run --rm -v ${PWD}:/local openapitools/openapi-generator-cli:latest-release generate -i /local/openapi.yaml -g python-fastapi -o /local`

**Comment by Minyus** [comment]
@wing328 
May I ask where I can find the pytest code generation template for python-fastapi?
I might be able to point out the problematic part.

**Comment by wing328** [comment]
You can find the templates in https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator/src/main/resources/python-fastapi.

**Comment by Minyus** [comment]
Thanks!
It seems the following line doesn't work well for POST or PUT endpoints (looks ok for GET or DELETE endpoints). 

https://github.com/OpenAPITools/openapi-generator/blob/d9f36cd44cfbceeb59e304ebb56a6ecc60723918/modules/openapi-generator/src/main/resources/python-fastapi/api_test.mustache#L24

For example, [this POST endpoint](https://github.com/OpenAPITools/openapi-generator/blob/d9f36cd44cfbceeb59e304ebb56a6ecc60723918/modules/openapi-generator/src/test/resources/3_0/petstore.yaml#L305) yields:

`body = openapi_server.Order()`

and causes the error while the following modification passes the pytest.

`body = Order().dict()`

**Comment by wing328** [comment]
cc @krjakbrjak (author of the fastapi generator)

**Comment by spacether** [comment]
The example comes from the bodyParam example property. Please search our Java code base to find out where bodyParam is assigned and to see how example is set

**Comment by MirUlr** [comment]
Also encounterd the problem described above. Is there any progress on that topic?
